README.md

```diff
@@ -56,6 +54,16 @@ The full list of changes is contained is available on the [release page](https:/
 * v2.0 is built against Roslyn 2.8.2, so will work against analyzers that use that version of Roslyn or earlier.
 * v2.0 uses NuGet v4.7, which supports the TLS1.3 security protocol.
 
+#### Updating Roslyn
+
+The SDK is compatible with analyzer targeting Roslyn from version 1.0.0 up to the version specified in [Directory.Build.props](https://github.com/SonarSource/sonarqube-roslyn-sdk/blob/master/Directory.Build.props#L9).
```

Comment:
There something wrong, when opening the file itself, the link does not work

Reply:
I don't see any issue when opening the link.
What is the issue on your side?

Reply:
Works now